docs/worker-node/install-wn-oasis.md

```diff
@@ -26,7 +26,7 @@ As with all OSG software installations, there are some one-time (per host) steps
 -   Ensure the host has [a supported operating system](../release/supported_platforms.md)
 -   On every worker node, [install and configure CVMFS](install-cvmfs.md)
 
-Once configured to use OASIS, grid jobs will download the worker-node software on demand (into the local disk cache).
+Once configured to use OASIS, OSPool jobs will download the worker-node software on demand (into the local disk cache).
```

Comment:
```suggestion
Once configured to use OASIS, OSPool jobs will download the worker-node software on demand (into the local disk cache).
```

Same thing as above, not limited to OSPool

Reply:
Fixed